17 add phq and mental diagnosis for postpartum psychosis

zma-wakehealth opened the pull request:
I updated the following:

1. added model for postpartum psychosis
2. rename columns of the preeclampsia models to match UNC's
3. added median features to be consistent with UNC's
4. added initial analysis for bp trajectory as requested by Prof. Page
5. added hypertension macro

2 and 3 will soon be updated according to UNC's new feature list.

mrhorvath commented on datamart/models/06_intermediate_reports/ammi/int_mental_cat.sql:
I would consider putting these category mappings into a seed file to make this more readable.

mrhorvath commented on datamart/models/07_reports/ammi/psychosis/rpt_psychosis_mental_cat_list.sql:
```diff
@@ -0,0 +1,23 @@
+        b.chief_complaint,
+        b.health_problem_list,
+        b.pcornet_defined_list
+    from mental_cat a
```
Generally disfavor using short aliases like 'a' or 'b'.  Using full names (or even better, don't alias them) makes code more readable.

mrhorvath reviewed (changes requested):
I think only one thing I'd really require, the condition should mark should pull from the staging.  The rest are suggestions.

Adding in some documentation for columns in the psychosis reports would be great.  See the schema.yml in the preeclampsia report layer.

zma-wakehealth commented:
Updated code as suggested:

1. change all reference to staging
2. put mental disease diagnosis into seed file and use join/union in the int_mental_cat.sql
3. added column definitions for the psychosis models
